doc/devel/color_changes.rst

```diff
@@ -0,0 +1,133 @@
+empirically bad color map and should not be the default color map.
+Due to the position that changing the appearance of plot is a back
+compatibility breaking change, this change has been put off for far
+longer than it should have been.  In addition to changing the default
+color map we plan to take the chance to change the default color-cycle
+on plots and to adopt a different color map for filled plots (``imshow``,
+``pcolor``, ``contourf``, etc) and for scatter like plots.
+
+
+Default Heat Map Colormap
+-------------------------
+
+The choice of a new color map is fertile ground to bike-shedding ("No,
+it should be _this_ color") so we have a proposed set criteria (via
+Nathaniel Smith) to evaluate proposed color maps.
+
+- it should be a sequential colormap, because diverging colormaps are
+  really misleading unless you know where the "center" of the data is,
+  and for a default colormap we generally won't.
+
+- it should be perceptually uniform, i.e., human subjective judgments
+  of how far apart nearby colors are should correspond as linearly as
+  possible to the difference between the numerical values they
+  represent, at least locally.
+
+- it should have a perceptually uniform luminance ramp, i.e. if you
+  convert to greyscale it should still be uniform. This is useful both
+  in practical terms (greyscale printers are still a thing!) and
+  because luminance is a very strong and natural cue to magnitude.
+
+- it should also have some kind of variation in hue, because hue
+  variation is a really helpful additional cue to perception, having
+  two cues is better than one, and there's no reason not to do it.
+
+- the hue variation should be chosen to produce reasonable results
+  even for viewers with the more common types of
+  colorblindness. (Which rules out things like red-to-green.)
+
+- For bonus points, it would be nice to choose a hue ramp that still
+  works if you throw away the luminance variation, because then we
+  could use the version with varying luminance for 2d plots, and the
+  version with just hue variation for 3d plots. (In 3d plots you
+  really want to reserve the luminance channel for lighting/shading,
+  because your brain is *really* good at extracting 3d shape from
+  luminance variation. If the 3d surface itself has massively varying
+  luminance then this screws up the ability to see shape.)
+
+- Not infringe any existing IP
+
+Example script
+++++++++++++++
+
+Proposed Colormaps
+++++++++++++++++++
+
+Default Scatter Colormap
+------------------------
+
+For heat-map like applications it can be desirable to cover as much of
+the luminence scale as possible, however when color mapping markers,
+having markers too close to white can be a problem.  For that reason
+we propose using a different (but maybe related) color map to the
+heat map for marker-based.  The design parameters are the same as
+above, only with a more limited luminence variation.
+
+
+Example script
+++++++++++++++
+::
```

Comment:
That was the plan eventually.  The goal is to get a gallery of this script run on all of the proposed color maps.